dbt/adapters/snowshupostgres/impl.py

```diff
@@ -9,7 +9,7 @@
 from dbt.adapters.snowshupostgres.connections import SnowshuPostgresConnectionManager
 
 
-# TODO reduce this qualification?
+# TODO find a better workaround for this
 # Repoint the macro name since the postgres version is inaccessible
 dbt.adapters.postgres.impl.GET_RELATIONS_MACRO_NAME = 'snowshupostgres_get_relations'
```

Comment:
this is less bad in my mind because we're explicitly naming it `snowshupostgres` - so finding what is basically a namespacing monkeypatch on the pg adapter is almost expected (or at least not unexpected), given the struggles with inheritance we've seen. If this was just a `snowshu` adapter this would feel more icky IMO.